packages/react-bindings/src/hooks/callStateHooks.ts

```diff
@@ -479,3 +479,43 @@ export const useIncomingVideoSettings = () => {
   );
   return settings;
 };
+
+/**
+ * Custom hook for toggling call recording in a video call.
+ *
+ * This hook provides functionality to start and stop call recording,
+ * along with state management for tracking the recording status
+ * and the loading indicator while awaiting a response.
+ */
+export const useToggleCallRecording = () => {
```

Comment:
I like the idea but let's move these utility hooks to some other file. This hook isn't a `callState` hook.

Reply:
I think there are a few more utility hooks like this in both SDKs that can be moved to bindings.

Reply:
I have created new file in the hooks folder named `callUtilsHooks.ts` and moved the `useToggleCallRecording` hook there.